bot/src/main/java/com/community/tools/service/github/DiscordGithubService.java

```diff
@@ -0,0 +1,20 @@
+package com.community.tools.service.github;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.apache.logging.log4j.util.Strings;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DiscordGithubService {
```

Comment:
It's really hard to understand the purpose of this service by its name

Reply:
Class renamed